eZ/Publish/Core/FieldType/User/Type.php

```diff
@@ -20,6 +24,17 @@
  */
 class Type extends FieldType
 {
+    /** @var \eZ\Publish\API\Repository\UserService */
+    protected $userService;
```

Comment:
It would be better to use SPI here.

User service, which reuses content service , will be calling field type for validation, so that we go full circle here and depend on user service is not particular well layered design.